feature-utils/poly-look/src/react-components/buttons/iconButton.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+import angleRight from "../../static/images/icons/angle-right.svg";
+import question from "../../static/images/icons/question.svg";
+import filter from "../../static/images/icons/filter.svg";
+
+import "./iconButton.css";
+
+/**
+ * Icon button. Has a filled variant.
+ * Any valid button HTML attributes are allowed.
+ * @param {Object} props
+ * @param {string} [props.icon] - Button icon
+ * Currently implemented options are: angleRight, question, filter.
+ * @param {string} [props.fillDirection] - This option adds a background color
+ * and a border radius to the direction specified.
+ * Available options are left and right.
+ */
+const IconButton = ({ icon, fillDirection = "", ...otherProps }) => {
+  const icons = {
+    angleRight,
+    question,
+    filter,
+  };
+
+  if (!icons[icon])
+    console.warn(
+      `IconButton: Invalid icon option. Available options: ${Object.keys(
```

Comment:
`icons` is apparently a `Set`. Maybe use it that way?

Reply:
I'm not convinced that it's worth it. For me this is pretty standard js. Using `Set` doesn't bring us any gains and it just makes the code look weird.

Reply:
OK, fair enough. I guess I like a bit of weirdness (as long as it's a bit mathematic-ish)